src/components/NoticeBox.js

```diff
@@ -1,13 +1,15 @@
 import React from "react";
 
 export function NoticeBox(props) {
-  if (props.errors.length == 0) {
+  if ( ! props.errors ) {
```

Comment:
This is marginally safer than checking `length`, because it won't error if props.errors isn't an array.